whois-nrtm4/src/main/java/net/ripe/db/nrtm4/persist/RpslObjectModel.java

```diff
@@ -0,0 +1,56 @@
+package net.ripe.db.nrtm4.persist;
+
+import net.ripe.db.whois.common.rpsl.ObjectType;
+import net.ripe.db.whois.common.rpsl.RpslObject;
+
+
+public class RpslObjectModel {
+
```

Comment:
check RpslObjectUpdateInfo bean, I think it's the same you are trying to use. We should re-use the exisiting beans and methods where we can.

Reply:
it's missing the `object` field which contains the RPSL. I've moved it to common and extended RpslObjectUpdateInfo instead.